src/plugin/customParseFormat/utils.js

```diff
@@ -0,0 +1,26 @@
+// handle negative values when overflowing month (handling positive and negative values)
+function modMonth(n, x) {
+  return ((n % x) + x) % x
+}
+
+// code duplication as we cannot use 'isLeapYear' plugin here;
+// 'isLeapYear' is only working with Dayjs objects
+function isLeapYear(year) {
+  return ((year % 4 === 0) && (year % 100 !== 0)) || (year % 400 === 0)
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export const daysInMonth = (year, month) => {
```

Comment:
we can use the built-in function 
```
dayjs(year + '-' + month).daysInMonth()
```

Reply:
sorry for the late reply, but somehow I missed this comment :-(

Of course we could use `dayjs(year + '-' + month).daysInMonth()` instead of my 'auxiliary' method.

But the problem is that we do not have the dayjs object available to the `checkOverflow()` function.

Therefore I created the `daysInMonth` method in the customParseFormat/utils file.  
As an additional bonus, this function runs faster than creating a new dayjs object (to be exact, more than 1 instance is created for getting daysInMonth).

What do you think about this?